CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/glyphs/FontInfo.java

```diff
@@ -59,7 +61,7 @@ public FontInfo(final String fontName, final int fontStyle, final int fontSize,
 
     private static Font getFont(final String fontName, final int fontStyle, final int fontSize) {
         Font font = null;
-        if (fontName.toLowerCase().endsWith(".otf") || fontName.toLowerCase().endsWith(".ttf")) {
+        if (StringUtils.endsWithIgnoreCase(fontName, FileExtensionConstants.OTF_EXTENSION) || StringUtils.endsWithIgnoreCase(fontName, FileExtensionConstants.TTF_EXTENSION)) {
```

Comment:
could `StringUtils.endsWithAny` be used here? (similar to what you've done in another change)

Reply:
I was hesitant because of case sensitivity of endsWithAny.

Reply:
Ok that's good to know